pages/administration/tools/booking-lookup.vue

```diff
@@ -0,0 +1,327 @@
+            nullify-value="performanceId"
+            :options="productions"
+        />
+        <UiInputSelect
+            v-else
+            v-model="productionSlug"
+            nullify-value="performanceId"
+            :disabled="true"
+            :options="[
+            { displayText: 'All', value: null }]"
+        />
+      </div>
+      <div>
+        <!-- Show the performances filter if we have any for a selected
+        production. If not, disable the filter.-->
+        <label>Performance</label>
+        <UiInputSelect
+            v-if="productionSlug && !$apollo.queries.performances.loading &&
+          performances.length > 0 && !disablePerformanceDropdown"
+            v-model="performanceId"
+            :options="performances"
+        />
+        <UiInputSelect
+            v-else
+            v-model="performanceId"
+            :disabled="true"
+            :options="[
+            { displayText: 'All', value: null }]"
+        />
+      </div>
+      <div>
+        <label>Status</label>
+        <UiInputSelect
+            v-model="bookingsStatus"
+            :options="[
+            { displayText: 'All', value: null },
+            { displayText: 'In Progress', value: 'IN_PROGRESS' },
+            { displayText: 'Paid', value: 'PAID' }
+          ]"
+        />
+      </div>
+    </div>
+    <UiCard class="mt-4">
+      <paginated-table
+          v-model:offset="bookingsOffset"
+          :page-info="bookingsPageInfo"
+          :items="bookings"
+          :max-per-page="10"
+          :loading="$apollo.queries.bookings.loading"
+          empty-text="No bookings found"
+      >
+        <template #head>
+          <table-head-item>Name</table-head-item>
+          <table-head-item>Quantity</table-head-item>
+          <table-head-item>Production</table-head-item>
+          <table-head-item>
+            Date<sort-icon
+              v-model="bookingsOrderBy"
+              :must-sort="false"
+              sort-field="start"
+          />
+          </table-head-item>
+          <table-head-item>Time</table-head-item>
+          <table-head-item>Status</table-head-item>
+          <table-head-item>
+            Created<sort-icon
+              v-model="bookingsOrderBy"
+              :must-sort="true"
+              sort-field="-createdAt"
+          />
+          </table-head-item>
+        </template>
+
+        <table-row
+            v-for="booking in bookings"
+            :key="booking.reference"
+            :striped="true"
+            :clickable="true"
+            @click="useRouter().push(`../productions/${booking.performance.production.slug}/bookings/${booking.reference}`)"
+        >
+          <table-row-item>
+            {{ booking.user.firstName }}
+            {{ booking.user.lastName }}
+            <p v-if="booking.creator.id !== booking.user.id" class="text-xs">
+              Created by {{ booking.creator.firstName }}
+              {{ booking.creator.lastName }}
+            </p>
```

Comment:
We show information about who created a booking (i.e. if they're comp tickets). Would it therefore also be useful to be able to filter by this with some tickbox?

Reply:
Currently this information is shown, but when I deal with [this](https://github.com/BristolSTA/uobtheatre-web/pull/375#discussion_r1820685028) below, this will effectively add said filter